meson.build

```diff
@@ -0,0 +1,32 @@
+project(
+    'fsps', 
+    'c',
+    version: run_command(
+        ['python3', '-m', 'setuptools_scm'],
+        check : true
+    ).stdout().strip(),
+    default_options: ['warning_level=0']
+)
+add_languages('fortran')
+
+# Python setup
+py_mod = import('python')
+py3 = py_mod.find_installation('python3')
+py3_dep = py3.dependency()
```

Comment:
You don't need to explicitly add the py3_dep to an `.extension_module()`, if you specify `meson_version: '>=0.63.0'` in the project function. This can remove a little bit of boilerplate.